Makefile

```diff
@@ -77,6 +81,7 @@ check: test lint-golint lint-codespell
 # Run tests
 test: generate fmt vet manifests
 	go test \
+	  -mod=readonly \
```

Comment:
Why do need this? `vendor/` directory is in the repository.

Reply:
Yes this needs to be removed.